core/pta/attestation.c

```diff
@@ -620,6 +628,8 @@ static TEE_Result cmd_hash_ta_memory(uint32_t param_types,
 	struct user_mode_ctx *uctx = NULL;
 	TEE_Result res = TEE_SUCCESS;
 	struct ts_session *s = NULL;
+	uint8_t *nonce_bbuf = NULL;
```

Comment:
This is the part which I think should be in `core/kernel/pseudo_ta.c` rather than here, I suppose in `copy_in_param()` / `update_out_param()`? Indeed this PTA also has issues in other commands, for example if `cmd_get_pubkey()` is invoked by a TA instead of a CA, and `CFG_PAN=y`, it will crash. @jenswi-linaro what do you think?